content/blog/2019/06/2019-06-25-Phase1-Updates-On-Working-Hours-Plugin

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "GSOC Phase 1 Updates On Working Hours Plugin"
+tags:
+- react
+- plugins
+- gsoc
+- gsoc2019
+author: shenjack
+---
+
+= UI Improvements with React on Working Hours Plugin
```

Comment:
The post would look better if there were longer paragraphs instead of 4 small ones